src/undo_log/undo_log.h

```diff
@@ -200,12 +200,6 @@ void UndoLog_Rollback
 	UndoLog log
 );
 
-// clears undo-log from all entries
-void UndoLog_Clear
-(
-	UndoLog log  // log to clear
-);
-
 // free undo-operation
 void UndoLog_FreeOp
```

Comment:
I think this function should be removed from the header and become a static function.